src/main/rx/ibus.c

```diff
@@ -117,7 +117,8 @@ uint8_t ibusFrameStatus(void)
 
     if (chksum == rxsum) {
         for (i = 0, pos = 2; i < IBUS_MAX_CHANNEL; i++) {
-            ibusChannelData[i] = ibus[pos++] + (ibus[pos++] << 8);
```

Comment:
maybe better to use (don't modify ibusChannelData twice):
```
ibusChannelData[i] = ibus[pos] + (ibus[pos+1] << 8); pos+=2;
```
or even
```
ibusChannelData[i] = ibus[i * 2 + 2] + (ibus[i * 2 + 3] << 8);
```
compiler is usually clever enough to generate optimal code in both cases ..